packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -372,7 +372,8 @@ async function handleDeactivated(
       tenantId: incomingPayment.tenantId,
       webhooks: finalizeWebhookRecipients(
         [incomingPayment.tenantId],
-        deps.config
+        deps.config,
+        incomingPayment.metadata
```

Comment:
I think we should have an additional column on the incoming payment, (maybe something like `initiatedBy: card | open-payments | admin` off the top of my head), so its auditable and we don't have to add anything to hide `metadata` fields when the incoming payment is fetched via Open Payments